pom.xml

```diff
@@ -26,10 +26,11 @@
 	<modelVersion>4.0.0</modelVersion>
```

Comment:
Global question about the different POMs: shouldn't they all use spaces instead of tabs as indents? Or the inverse?

Reply:
I set that in `.editorconfig`. `pom.xml` files are using tab. We could change that.